tests/vmi_multus_test.go

```diff
@@ -1342,11 +1346,10 @@ func defaultCloudInitNetworkData() string {
 	return networkData
 }
 
-func cloudInitNetworkDataWithStaticIPsByMac(nicName, macAddress, ipv4Address string) string {
+func cloudInitNetworkDataWithStaticIPsByMac(nicName, macAddress string, ipAddress string) string {
```

Comment:
nit: maybe worth doing this ?
`nicName, macAddress, ipAddress string`

Reply:
Done